packages/cadl-rlc-emitter/test/integration/generated/authentication/apiKey/package.json

```diff
@@ -7,13 +7,13 @@
   "keywords": ["node", "azure", "cloud", "typescript", "browser", "isomorphic"],
   "license": "MIT",
   "main": "dist/index.js",
-  "module": "./dist-esm/index.js",
+  "module": "./dist-esm/src/index.js",
```

Comment:
do you generate test or example for this test case ? if no, why there's a src here ? Please make sure this file is truely there after the build.

Reply:
Yes we have build verification in cadl-rlc-test folder and they do pass the build process